src/blueprints/timeline-state-resolver-types/telemetrics-type.ts

```diff
@@ -0,0 +1,14 @@
+import { TimelineObject } from '../../model/entities/timeline-object'
+import { DeviceType } from '../../model/enums/device-type'
+
+export interface TelemetricsCallPresetTimelineObject extends TimelineObject {
+  content: {
+    deviceType: DeviceType.TELEMETRICS
+    type: TelemetricsType.CALL_PRESET
+    presetShotIdentifier: number[]
```

Comment:
I saw that value is `number[]`, but the name looks like is only one value `number`

Reply:
There is a reason for this and it is currently not obvious in the current state of the code (it is the plan to make it obvious once we get to that part of the refactor).

There is something in the Sofie ecosystem called Timeline State Resolver (TSR for short). TSR is responsible for converting our TimelineObjects into actual commands for the different hardware devices. In order for TSR to do this, we must provide data in a structure that TSR knows. 

The `content` object above is defined in the TSR repository (https://github.com/tv2/tv-automation-state-timeline-resolver/blob/develop/packages/timeline-state-resolver-types/src/telemetrics.ts) and we can't deviate from that. If we do, TSR won't be able to translate our data. 

So since TSR currently define `presetShotIdentifier` as `number[]`, we have to do so as well. (We also have to name it the same).

As mentioned earlier, it is the plan to expose types so we don't have to manually maintain the types. But that is currently out of scope.

Reply:
I understand